cosmetics-web/app/controllers/responsible_persons/delete_formulation_file_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ResponsiblePersons::DeleteFormulationFileController < SubmitApplicationController
+  before_action :set_notification, :set_component
```

Comment:
Seems `#set_notification` is redundant in this controller, given that `#set_component` overrides the notification instance var set and authorisation.
```suggestion
  before_action :set_component
```